isos/bootstrap/bootstrap

```diff
@@ -58,24 +58,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
         # copy the init binary into the new root - ideally I'd run this purely from memory without a presence on the filesystem
         cp /bin/tether ${MOUNTPOINT}/.tether/tether-bin
 
-        # temporary hack for waiting the network
-        while ! ip addr show eth1 | grep " inet "; do sleep 1;done
-
         cat << EOF > ${MOUNTPOINT}/.tether/tether
 #!/bin/sh
 
-if [ -x /sbin/ip -o -x /bin/ip ]; then
-    addr=$(ip addr show eth1  | grep " inet " | cut -d " " -f6)
-    gateway=$(ip route show | grep "^default" | cut -d " " -f3)
-
-    echo "Setting IP addr to " \$addr > /dev/ttyS1
-    echo "Setting route to" \$gateway > /dev/ttyS1
-
-    ip addr add \$addr dev eth1
-    ip link set dev eth1 up
-    ip route add default via \$gateway dev eth1
-fi
-
 /.tether/tether-bin
```

Comment:
Now that we have lib/etcconf/hosts.go, I think prefer tether writing the default config files and then dropping this shell wrapper and calling the binary directly.